src/components/Editor.vue

```diff
@@ -650,7 +652,14 @@ export default {
 		},
 
 		onSave() {
-			emit('files:file:updated', { fileid: this.fileId })
+			const user = getCurrentUser()
+			const node = new File({
+				id: this.fileId,
+				source: generateRemoteUrl(`dav/files/${user.uid}${this.relativePath}`),
+				mtime: new Date(),
+				mime: 'text/markdown',
```

Comment:
The text app might also handle other mime types, so we should take the one that the viewer passes along to us

Reply:
@luka-nextcloud Can you check that one please?